mpsl/CHANGELOG.rst

```diff
@@ -31,7 +31,7 @@ Added
 * On the nRF54L15 SoC, Errata 39 is now applied.
 * Added the defines :c:macro:`MPSL_TIMER0` to clarify which timer is being used by MPSL and the timeslot implementation. (DRGN-24434)
 * New API provided for integration with an external power management system.
-  The API is only support on nRF54H Series SoCs.
+  The API is only supported on nRF54H Series SoCs.
```

Comment:
Could you drop these changes in mpsl/CHANGELOG.rst? I assume it's better if we fix it from our side and push the changes with a nrfxlib update

Reply:
Done.